core/src/main/java/org/apache/accumulo/core/client/AccumuloSecurityException.java

```diff
@@ -182,7 +181,7 @@ public String getMessage() {
     StringBuilder message = new StringBuilder();
     message.append("Error ").append(errorCode);
     message.append(" for user ").append(user);
-    if (!StringUtils.isEmpty(tableInfo)) {
+    if (!tableInfo.isEmpty()) {
```

Comment:
These aren't equivalent. The `StringUtils` version checks for `null` and `""` in one call.

Reply:
Good point.  I thought the new "isBlank()" in Java 11 would allow removal of the dependency.  It is probably better to just upgrade to lang3.